app/models/defence_request.rb

```diff
@@ -10,8 +10,14 @@ class DefenceRequest < ActiveRecord::Base
   scope :not_draft, -> { where.not(state: "draft") }
   scope :ordered_by_created_at, -> { order(created_at: :asc) }
 
-  def self.has_solicitor(solicitor)
-    where(solicitor_uid: solicitor.uid)
+  class << self
```

Comment:
What is the motivation for switching class method syntax here? This would, for instance, raise issues with Rubocop or other style checkers, adds noise to the class and seems a bit spurious?